user.php

```diff
@@ -255,8 +256,9 @@
         <h3>Info</h3>
         <hr>
 
-        <?php if( isset($_SESSION['loggedin']) && $_SESSION['loggedin'] === 'true' && $_SESSION['login'] === $nick  ) {
-          echo "<h4>Congratulations, you have logged in.  You have earnt one reputation point.  Account management and more <a target=\"_blank\" href=\"https://melvincarvalho.gitbooks.io/gitpay/content/\">features</a> coming soon.<h4><hr>";
+        <?php
+        if( isset($_SESSION['loggedin']) && $_SESSION['loggedin'] === 'true' && $_SESSION['login'] === $nick  ) {
+          echo "<h4>Congratulations, you have logged in.  You have earnt one reputation point.  Account management and more <a target=\"_blank\" href=\"https://melvincarvalho.gitbooks.io/gitpay/content/\">features</a> coming soon.<h4><h4>If you wish to opt-out of gitpay, click here: <a href='/deleteuser.php'>Remove my account</a></h4><hr>";
```

Comment:
I would totally add here a confirmation step (non-revokable action) and an URL that doesn't leak the extension:

```html
<a class="delete" href="/deleteuser">...
```

```js
document.querySelector('a.delete').addEventListener('click', function(e) {
  var result = confirm("Are you sure?");
  if (!result) e.preventDefault();
});
```